### PR TITLE
Add OfficeToPDF

### PR DESCRIPTION
**Pull request by abloemert**

### CODE REVIEW

**Comment by abloemert** [comment]
@mariusvniekerk, can this be merged?

File: recipes/officetopdf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/cognidox/{{ name }}/releases/download/v{{ version }}/{{ name }}.exe
+  sha256: {{ hash }}
+
+build:
+  skip: True  # [not win]
+  number: {{ build }}
+  script: "copy {{ name }}.exe %LIBRARY_BIN%"
```
Comment (mariusvniekerk) [inline recipes/officetopdf/meta.yaml]: erm... could we compile this instead?  I'm not very keen on repacking exes

Comment (abloemert) [reply recipes/officetopdf/meta.yaml]: good suggestion, I'll try to achieve that!

File: .appveyor.yml [hunk 2]
```diff
@@ -51,6 +51,5 @@ test_script:
 
     # copy any newly created conda packages into the conda_packages dir
     - cmd: mkdir conda_packages
-    # Uncomment the following two lines to make any conda packages created
```
Comment (chrisburr) [inline .appveyor.yml]: Can you revert these changes as they're outside of the recipe directory.

File: recipes/officetopdf/LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+Apache
```
Comment (chrisburr) [inline recipes/officetopdf/LICENSE.md]: This file appears to be included in the sources that are downloaded. I think you can delete it and `conda-build` will automatically find it from there instead. This ensures it will never be outdated if the license was to change.
